tests/st/utils/data.py

```diff
@@ -289,6 +289,25 @@
     }
 }
 
+hostendpoint_name1_rev3 = {
+    'apiVersion': API_VERSION,
+    'kind': 'HostEndpoint',
+    'metadata': {
+        'name': 'endpoint1',
+        'labels': {'type': 'frontend', 'misc': 'version1'},
+        'annotations': {'key': 'value'},
+        'selfLink': 'test-self-link',
```

Comment:
I think this should include uuid and creation timestamp to check that the values in the YAML are not stored as well.